src/models/pickup.js

```diff
@@ -32,6 +32,6 @@ export default class Pickup extends EasyPostObject {
   lowestRate(carriers, services) {
     const rates = this.pickup_rates || [];
 
-    return Utils.getLowestRate(rates, carriers, services);
+    return new EasyPostClient('fake_api_key').Utils.getLowestRate(rates, carriers, services);
```

Comment:
issue: we should not be constructing clients inside our own code. The client is for the user. Because we have direct access to the code before it gets compiled, we should still be able to reference the utils relatively. It's our users who cannot do this because they only get access to the single exposed client.